Os/Posix/IPCQueue.cpp

```diff
@@ -238,23 +238,23 @@ namespace Os {
     }
 
     NATIVE_INT_TYPE IPCQueue::getQueueSize() const {
-        QueueHandle* queueHandle = (QueueHandle*) this->m_handle;
+        QueueHandle* queueHandle = reinterpret_cast<QueueHandle*>(this->m_handle);
         mqd_t handle = queueHandle->handle;
 
         struct mq_attr attr;
         int status = mq_getattr(handle, &attr);
         FW_ASSERT(status == 0);
-        return (U32) attr.mq_maxmsg;
+        return static_cast<U32>(attr.mq_curmsgs);
```

Comment:
Why did the member change here?

Reply:
Oh yikes, copy and paste error I somehow overlooked. How did this not break the unit tests.